src/components/OnBoard/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC } from 'react';
+import { SwiperSlide } from 'swiper/react';
+import { Pagination } from 'swiper/modules';
+import 'swiper/css';
+import 'swiper/css/pagination';
+import { CustomSwiper } from './styled';
```

Comment:
create a new component for swiper and move all logic and markup related to swiper to this component and just use it here